docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -276,6 +349,45 @@ RETURN count(*);
 
 Copy and paste the contents of the script into a file called `import.cypher`.
 
+[WARNING]
+====
+Mandrel 22.0.0 contains a bug where the symbolic links used by the import cypher file are not correctly set when generating reports within a container
```

Comment:
This looks like a good reason to use 21.3.x.x instead.

Reply:
Hmmm, it's a very small issue that can easily be worked around. I don't agree.